packages/side-nav/src/SideNavGroup.tsx

```diff
@@ -168,36 +194,81 @@ function SideNavGroup({
   children,
   collapsible = false,
   initialCollapsed = true,
+  glyph,
+  className,
   ...rest
 }: SideNavGroupProps) {
   const [open, setOpen] = React.useState(!initialCollapsed);
   const nodeRef = React.useRef(null);
   const ulRef = React.useRef<HTMLUListElement>(null);
   const { usingKeyboard: showFocus } = useUsingKeyboardContext();
+  const { currentPath } = useSideNavContext();
+  const menuGroupLabelId = useMemo(() => menuGroupIdAllocator.generate(), []);
+  const menuId = useMemo(() => menuGroupIdAllocator.generate(), []);
+
+  const isActiveGroup: boolean = useMemo(() => {
+    return React.Children.toArray(children).some(child => {
+      return (
+        isComponentType(child, 'SideNavItem') &&
+        (child.props.active || child.props.href === currentPath)
+      );
+    });
+  }, [children, currentPath]);
+
+  const accessibleGlyph =
+    (glyph && isComponentType(glyph, 'Glyph')) || isComponentType(glyph, 'Icon')
```

Comment:
I think there's a reason why this isComponentType won't work on our glyphs (I don't remember what it is) but I know that we exposed the .isGlyph property on glyphs for a reason

Reply:
Probably because you need to do this hah. Regardless, this is working as intended,

Reply:
Really? With an Icon and a Glyph?

Reply:
You're right. Fixed this